src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.uwp.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.ComponentModel;
+using System.Linq;
+using Windows.UI.Core;
+using Windows.UI.Input.Inking;
+using Windows.UI.Xaml.Controls;
+using Xamarin.CommunityToolkit.UI.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.UWP;
+using Size = Windows.Foundation.Size;
+
+[assembly: ExportRenderer(typeof(DrawingView), typeof(DrawingViewRenderer))]
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class DrawingViewRenderer : ViewRenderer<DrawingView, InkCanvas>
+	{
+		InkCanvas canvas;
+		InkDrawingAttributes inkDrawingAttributes;
+		bool disposed;
+
+		protected override void OnElementChanged(ElementChangedEventArgs<DrawingView> e)
+		{
+			base.OnElementChanged(e);
+			if (Control == null && Element != null)
+			{
+				canvas = new InkCanvas();
+				inkDrawingAttributes = new InkDrawingAttributes
+				{
+					Color = Element.LineColor.ToWindowsColor(),
+					Size = new Size(Element.LineWidth, Element.LineWidth)
+				};
+				canvas.InkPresenter.UpdateDefaultDrawingAttributes(inkDrawingAttributes);
+				canvas.InkPresenter.InputDeviceTypes = CoreInputDeviceTypes.Mouse |
+													   CoreInputDeviceTypes.Pen |
+													   CoreInputDeviceTypes.Touch;
```

Comment:
This seems a bit weird formatting, but might be just GitHub

Reply:
Looks like that's a github issue